Makefile

```diff
@@ -52,7 +52,7 @@ PUBLISH ?= onTag
 # Main targets
 .PHONY: start
 start:
-	@NODE_ENV=$(NODE_ENV) DEV_SERVER=$(DEV_SERVER) npx electron .
+	@NODE_ENV=$(NODE_ENV) DEV_SERVER=$(DEV_SERVER) npx electron . --inspect
```

Comment:
Did you intend to leave this debugging in for `start` and not just `dev`? Not sure if I am reading it right.

Reply:
I left it there on purpose.

Reply:
why?

also why did we remove `rebuild-deps`?